### ISSUE TITLE
CSS trivia questions | Spanish Translation

### ISSUE THREAD

**Issue by hectoraldairah** [yangshun/front-end-interview-handbook#365]
Hello! I'd like to translate de CSS trivia questions into Spanish

Can I work with that?

**Comment by yasaminashoori** [comment]
> Hello! I'd like to translate de CSS trivia questions into Spanish
> 
> Can I work with that?

Yes mate ✌, as yangshun mentioned you can add your translation following these instructions: 
[https://github.com/yangshun/front-end-interview-handbook/blob/main/CONTRIBUTING.md](url)
